homeassistant/components/sensor/simulated.py

```diff
@@ -0,0 +1,141 @@
+"""
+Adds a simulated sensor.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.simulated/
+"""
+import datetime as datetime
+import math
+import random
+import logging
+
+import voluptuous as vol
+
+from homeassistant.helpers.entity import Entity
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_NAME
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+
+_LOGGER = logging.getLogger(__name__)
+SCAN_INTERVAL = datetime.timedelta(seconds=1)
+ICON = 'mdi:chart-line'
+
+CONF_UNIT = 'unit'
```

Comment:
Can we name this unit_of_measurement ?

Reply:
What do you propose? It’s just a number after all, not a measurement of a physical quantity